content/blog/2024-04-04-tmix-03.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "This Month in Xilem, March 2024"
+authors = ["Daniel McNab"]
++++
+
+<!-- I don't think we need to set the date, as it is inferred from the slug -->
+Since the last update, there have been three newly released projects from Linebender:
+
+- [![Velato's version on crates.io.](https://img.shields.io/crates/v/velato.svg)](https://crates.io/crates/velato)
+ [Velato][]: A parser and driver for Lottie animation files in applications using Vello
+- [![Vello SVG's version on crates.io.](https://img.shields.io/crates/v/vello_svg.svg)](https://crates.io/crates/vello_svg)
+  [Vello SVG][]: Allows displaying (some) SVG documents in Vello applications
+- [![Tracing Android Trace's version on crates.io.](https://img.shields.io/crates/v/tracing_android_trace.svg)](https://crates.io/crates/tracing_android_trace)
+  [Android Trace][]: Resilient bindings to [NDK Tracing][] and an integration with [tracing][]
```

Comment:
Interesting, I wonder why it can't deal with `[note]`.

Anyway, I'm not bothered by it enough to outweigh your utility of it, so I'm fine with keeping them.

Reply:
I suspect it's because there are some cases where you'd actually want to use `[some text]`, e.g. as an aside?

But yeah, I'm not sure. Great